mbus/mbus-tcp.c

```diff
@@ -81,6 +81,9 @@ mbus_tcp_connect(mbus_handle *handle)
         return -1;
     }
 
+    // Free addr info, we do not need it anymore
+    freeaddrinfo(servinfo);
```

Comment:
Unfortunately this seems the wrong place. In the error case above this should be freed, too. Please move it above.